client/src/pages/ContactUsPage/ContactUsPage.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { Typography, Box, Grid, Link, Stack, styled } from "@mui/material"
+import FacebookIcon from '@mui/icons-material/Facebook';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import YouTubeIcon from '@mui/icons-material/YouTube';
+import PinterestIcon from '@mui/icons-material/Pinterest';
+import TelegramIcon from '@mui/icons-material/Telegram';
+import ContactsForm from '../../features/contactUs/components';
+
+const icons = [
+  <FacebookIcon />,
+  <TwitterIcon />,
+  <LinkedInIcon />,
+  <YouTubeIcon />,
+  <PinterestIcon />,
+  <TelegramIcon />
+]
+
+const SocialsContainer = styled(Box)(() => ({
+  display: 'flex',
+  flexDirection: 'row'
+}))
+
+const SocialItem = styled('button')(() => ({
+  backgroundColor: 'rgba(40, 44, 52, 0.2)',
+  margin: '5px',
+  marginLeft: 0,
+  width: '40px',
+  height: '40px',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  borderRadius: '10px',
+  cursor: 'pointer',
+  transition: 'all 0.3s ease',
+  border: 'none',
+  '&:hover': {
+    backgroundColor: 'rgba(0, 75, 252, 0.04)'
+  }
+}))
+
+const StackContact = styled((props) => <Stack {...props} />)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column', 
```

Comment:
i think they are there by default in the stack. no need to repeat

Reply:
done in new commit